Clarify how shading in popover works (#730)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxPopover/1 Configuration/shading.md

type: Boolean
default: false
---

> Shading applies to the first element with a <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/position" target="_blank">position</a> value different from `static`.

Reviewer: Shading applies or shading is applied?

> Shading applies to the element specified by the [position](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) option. If this element is not specified, shading applies to the **window**.

Reviewer 2: In the [position](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position
) topic we write that the position.of is connected with the target option. The latter has higher priority
@ksercs  should we mention both options here?

Reviewer 3: Maybe, it's good idea to write that we are talking about `position.of` of the popover root element (not target)

Reviewer 3: Popover shading is not connected with `target` option/element despite what is written in the documentation

> Shading applies to the element specified[position](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) option or to the **window** if the target element for the **position**.**of** option is not specified.

Reviewer 4: The sentence is too long. It is better to split it in two.

> Shading applies to the element specified[position](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) option. If the target element for the **position**.**of** option is not specified, shading applies to the **window** 

Reviewer 4: ```suggestion
Shading applies to the element specified by the [position](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) option. If this element is not specified, shading applies to the **window**.
```

type: Boolean
default: false
---

> Shading applies to the first element with non-static <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/position" target="_blank">position</a> in terms of CSS. The sequence is the following: [container](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#container) => Descendant elements => **window**.

Reviewer 4: I went to the MDN document, searched for "non-static", and found nothing. Looks like we should a better job explaining this.

Author: There are five types of position: 
1. Static
2. Relative
3. Fixed
4. Absolute
5. Sticky

This is quite a basic thing. I'm sure that web devs have at least minimal understanding of this. 

The list is on the first screen (I don't know why the fixed type is omitted).
![image](https://user-images.githubusercontent.com/33040934/81416955-25295400-9153-11ea-9c27-3121c6d45459.png)

Reviewer 4: The word "static" rings a bell, "non-static" doesn't. We better rephrase it.

BTW, it's not entirely clear what you mean by "the sequence". Is that the order in which the elements are checked for the position value?

> The [target](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#target) option does not affect shading.

Reviewer 4: Why mention this?
